app/lib/actions/candlepin/import_pool_subscription_handler.rb

```diff
@@ -16,7 +16,7 @@ def content
       end
     end
 
-    class ReindexPoolSubscriptionHandler
+    class ImportPoolSubscriptionHandler
```

Comment:
Nitpick: This class only seems to deal with pools (I don't see a mention of subscriptions), I'd change the name to reflect this is just about importing pools.